### ISSUE TITLE
loss分母是不是还需要平方

### ISSUE THREAD

**Issue by cuppersd** [xiaomaxiao/PSENET#8]
loss = K.mean((2.0 * intersection + smooth ) / (K.sum(y_true,axis=(1,2,3)) + K.sum(y_pred,axis=(1,2,3)) + smooth))
您好，和原始论文相比还差个平方？

**Comment by xiaomaxiao** [comment]
嗯 这个是我修改的地方 没有用平方
